### PR TITLE
Interactions: Collapse child interactions

### PR DESCRIPTION
**Pull request by ghengeveld**
Issue:
AP-1856

This collapses child interactions on `waitFor` and provides a toggle icon to reveal them.

https://user-images.githubusercontent.com/321738/173844722-4aaf0a08-89c0-4110-9cab-4c6ef6f49f93.mov

## How to test

- [x] Is this testable with Jest or Chromatic screenshots?

### CODE REVIEW

File: addons/interactions/src/components/Interaction/Interaction.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { IconButton, Icons, TooltipNote, WithTooltip, Bar } from '@storybook/components';
```
Comment (shilman) [inline addons/interactions/src/components/Interaction/Interaction.tsx:2]:
```suggestion
import { IconButton, Icons, TooltipNote, WithTooltip } from '@storybook/components';
```
Fix deepscan

**Review by shilman — APPROVED** [review]
So nice @ghengeveld 😍

**Comment by ghengeveld** [comment]
@shilman Ready to merge when the checks are green (save for cra-bench).

**Comment by yannbf** [comment]
@ghengeveld could you update the mocks so we have stories for the collapse scenario? thanks!

**Pull request merged**